google/cloud/bigquery/v2/minimal/internal/job_idempotency_policy.cc

```diff
@@ -43,6 +43,10 @@ Idempotency BigQueryJobIdempotencyPolicy::InsertJob(InsertJobRequest const&) {
   return Idempotency::kNonIdempotent;
 }
 
+Idempotency BigQueryJobIdempotencyPolicy::CancelJob(CancelJobRequest const&) {
+  return Idempotency::kNonIdempotent;
```

Comment:
I would think canceling a job is idempotent.  You can only succeed at canceling the job once?